website/docs/language/state/locking.mdx

```diff
@@ -14,7 +14,7 @@ others from acquiring the lock and potentially corrupting your state.
 State locking happens automatically on all operations that could write
 state. You won't see any message that it is happening. If state locking fails,
 Terraform will not continue. You can disable state locking for most commands
-with the `-lock` flag but it is not recommended.
+with the `-lock=false` flag but it is not recommended.
 
 If acquiring the lock is taking longer than expected, Terraform will output
 a status message. If Terraform doesn't output a message, state locking is
```

Comment:
```suggestion
If acquiring the lock takes longer than expected, Terraform outputs
a status message. If Terraform does not output a message, state locking is
```

Suggest changing "is taking" to "takes" to use fewer words. Suggest changing "will output" to "outputs" to avoid future tense. Suggestion changing "doesn't" to "does not" to avoid using contractions.